### PR TITLE
Add JavaScript compilation step

### PR DESCRIPTION
**Pull request by aduth**
**Why**: For improved maintainability as we seek to expand JavaScript implementation.

Slack context: https://gsa-tts.slack.com/archives/CR6BF3CQK/p1647960953159559?thread_ts=1647898836.557379&cid=CR6BF3CQK

### CODE REVIEW

**Review by zachmargolis — APPROVED** [review]
LGTM! 🚀

**Comment by aduth** [comment]
Looking good again on the analytics page after latest round of changes:

https://federalist-7d0b2b76-42fc-4df1-9825-8c3cd77e0a15.app.cloud.gov/preview/18f/identity-handbook/aduth-js-compilation/articles/analytics-events.html

File: assets/js/analytics-events.tsx [hunk 1]
```diff
@@ -0,0 +1,214 @@
+import { h, render, Fragment } from "preact";
+import { urlify } from "anchor-js";
```
[the remark this answers is not in the document]
Comment (aduth) [reply assets/js/analytics-events.tsx]: Too bad it was broken. 😆  The fixed version isn't quite as nice, but better than what we had:

https://github.com/18F/identity-handbook/blob/f6e7756f7b06925a13ee97645323d4ea5d5f200b/assets/js/analytics-events.tsx#L2-L5

[the remark this answers is not in the document]
Comment (aduth) [reply assets/js/analytics-events.tsx]: There is a [DefinitelyTyped package for `anchor-js`](http://npmjs.com/package/@types/anchor-js), but apparently `urlify` isn't included in it (maybe not deemed part of its "public" interface):

https://unpkg.com/browse/@types/anchor-js@4.2.1/index.d.ts

**Pull request merged**